From f2151fa730058a1baf23ed3dc082c91df6351da1 Mon Sep 17 00:00:00 2001
From: "Zeyi (Rice) Fan" <zeyi@fb.com>
Date: Fri, 4 Feb 2022 15:17:53 -0800
Subject: [PATCH] fix build failure on Windows

Summary: Fix a build error on Windows where `std::mismatch` is not found.

Reviewed By: xavierd

Differential Revision: D34012963

fbshipit-source-id: 9f5aa21d03a92fdadf5bc83943a159a2f7872144
---
 thrift/compiler/generate/build_templates.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/thrift/compiler/generate/build_templates.cc b/thrift/compiler/generate/build_templates.cc
index 56f13e8dde8..acd9dd30a75 100644
--- a/thrift/compiler/generate/build_templates.cc
+++ b/thrift/compiler/generate/build_templates.cc
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+#include <algorithm>
 #include <fstream>
 #include <iostream>
 #include <iterator>
